app/src/pages/explorer/dataset/Dataset.vue

```diff
@@ -52,11 +54,12 @@
 
               <div class="section_md-header u_display-flex image-detail-page__relatedImg">
                 <md-card
-                  v-for="(item, index) in dataset[datasetFields['distribution']]"
+                  v-for="(item, index) in distributions"
                   class="md-card-class u--margin-none"
                   :class="`charts-${index+1} charts-${index+1}-narrow`"
                   :key="`card_${index}`"
                 >
+                <a :href="item?.downloadLink">
```

Comment:
I'll suggest using the optional chaining util function to add # just in case this comes back as undefined